File: scripts/fixtures/code-probe/UserService.cs
using System;
using System.Collections.Generic;
using static System.Math;

namespace Example.Inventory;

/// <summary>Coordinates inventory operations.</summary>
public class UserService
{
    private readonly string region;

    public UserService(string region)
    {
        this.region = region;
    }

    public double Round2(double value) => Round(value, 2);
}

public interface IAuditSink
{
    void Record(string evt);
}

public enum Severity
{
    Low,
    High
}

public struct Point
{
    public int X;
    public int Y;
}

File: scripts/fixtures/code-probe/UserService.java
package com.example.inventory;

import java.util.List;
import java.util.Map;
import static java.util.Collections.emptyList;

/** Coordinates inventory operations for a single warehouse. */
public class UserService {
    private final String region;
    private final Map<String, Integer> balances;

    public UserService(String region, Map<String, Integer> balances) {
        this.region = region;
        this.balances = balances;
    }

    public List<String> lowStock(int threshold) {
        return emptyList();
    }
}

interface AuditSink {
    void record(String event);
}

enum Severity {
    LOW,
    HIGH
}
